src/components/Delegate/StakingOpportunities.tsx

```diff
@@ -0,0 +1,52 @@
+import { ArrowForwardIcon } from '@chakra-ui/icons'
+import { Button, HStack, Stack } from '@chakra-ui/react'
+import { caip2, CAIP19 } from '@shapeshiftoss/caip'
+import { ChainTypes } from '@shapeshiftoss/types'
+import { NavLink } from 'react-router-dom'
+import { Card } from 'components/Card/Card'
+import { Text } from 'components/Text'
+import { selectAssetByCAIP19 } from 'state/slices/assetsSlice/assetsSlice'
+import { AccountSpecifier } from 'state/slices/portfolioSlice/portfolioSlice'
+import { useAppSelector } from 'state/store'
+
+import { StakingOpportunitiesRow } from './StakingOpportunitiesRow'
+
+type StakingOpportunitiesProps = {
+  tokenId?: string
+  assetId: CAIP19
+  accountId?: AccountSpecifier
+  isLoaded?: boolean
+}
+
+export const StakingOpportunities = ({ assetId: caip19 }: StakingOpportunitiesProps) => {
+  // TODO: wire up with real validator data
+  const asset = useAppSelector(state => selectAssetByCAIP19(state, caip19))
+  const { chain } = caip2.fromCAIP2(asset.caip2)
+  const validators = [{ name: 'Cosmos Validator' }]
+
+  // TODO: remove this and only show for cosmos sdk chains
+  if (chain === ChainTypes.Ethereum || chain === ChainTypes.Bitcoin) return null
```

Comment:
i feel like this will become a common pattern. do you wanna create a conditional render type hook, where it's like 

```
return renderForChains([ChainTypes.Cosmos], () => <div>foo</div>)
```

Reply:
In order for that to work, it would need a reference chain (from the current asset) to compare to. Maybe something like this:
```
return renderForChains([ChainTypes.Cosmos], referenceChain, () => <div>foo</div>)
```